README.md

```diff
@@ -193,13 +193,13 @@ populated from the query params.
 The Search plugin comes with a set of predefined search filters that allow you to
 easily create the search results you need. Use:
 
-- ``value`` to limit results to exact matches
-- ``like`` to produce results containing the search query (``LIKE`` or ``ILIKE``)
-- ``boolean`` to limit results by truthy (by default: 1, true, '1', 'true', 'yes', 'on') and falsy (by default: 0, false, '0', 'false', 'no', 'off') values which are passed down to the ORM as true/1 or false/0 or ignored when being neither truthy or falsy.
-- ``finder`` to produce results using a [(custom)](http://book.cakephp.org/3.0/en/orm/retrieving-data-and-resultsets.html#custom-find-methods) finder
-- ``compare`` to produce results requiring operator comparison (
+- ``Value`` to limit results to exact matches
+- ``Like`` to produce results containing the search query (``LIKE`` or ``ILIKE``)
+- ``Boolean`` to limit results by truthy (by default: 1, true, '1', 'true', 'yes', 'on') and falsy (by default: 0, false, '0', 'false', 'no', 'off') values which are passed down to the ORM as true/1 or false/0 or ignored when being neither truthy or falsy.
```

Comment:
I would use ` instead of ' for all code pieces here to not confuse `false` with string `'false'` for example.